e2e-tests/portal/cypress/fixtures/deal.js

```diff
@@ -6,8 +6,8 @@ const CONSTANTS = require('./constants');
 const deal = {
   submissionType: CONSTANTS.DEALS.SUBMISSION_TYPE.MIN,
   updatedAt: Date.now(),
-  bankInternalRefName: 'DTFS2-2815 MIN - pre submit',
-  additionalRefName: 'DTFS2-2815 MIN - pre submit',
+  bankInternalRefName: '123',
+  additionalRefName: 'BssDeal',
```

Comment:
I think this would be cleaner: 

```suggestion
  bankInternalRefName: 'Mock additional ref name',
  additionalRefName: 'Mock additional ref name',
```

Reply:
I renamed as suggested but when running one of the test fails with assertion error as expected `Clone 123` but received `Clone Mock additional ref name`

Reply:
abandon-a-deal test fails if changed as suggested